ci-operator/step-registry/ipi/conf/ipi-conf-commands.sh

```diff
@@ -35,6 +35,21 @@ fips: true
 EOF
 fi
 
+if [ -n "${CLUSTER_NETWORK_CIDR:-}" ] || [ -n "${CLUSTER_NETWORK_HOST_PREFIX:-}" ] || [ -n "${MACHINE_NETWORK_CIDR:-}" ] || [ -n "${SERVICE_NETWORK_CIDR:-}" ]; then
```

Comment:
@mehabhalodiya @mehabhalodiya This pr needs review and approval by https://github.com/openshift/release/blob/master/ci-operator/step-registry/ipi/OWNERS. 
@jianlinliu Please help to review this pr especially this file, as it has a big impact.

Reply:
It deserved to develop a new step to patch install-config.yaml instead of update it directly here to avoid a large impact on all the ci jobs.

Reply:
@jianlinliu @mehabhalodiya  We're referencing the usage of ref `ovn-conf-custom-cidr` in this chain https://github.com/openshift/release/blob/master/ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/ovn/custom-cidr/provision/cucushift-installer-rehearse-aws-ipi-ovn-custom-cidr-provision-chain.yaml
```
chain:
  as: cucushift-installer-rehearse-aws-ipi-ovn-custom-cidr-provision
  steps:
  - chain: ipi-conf-aws
  - ref: ovn-conf-custom-cidr
....
```
I think there should be an existing step in `chain: ipi-conf-aws` that already patches install-config.yaml.